package.json

```diff
@@ -1,39 +1,65 @@
 {
```

Comment:
example of the `format-package` script that organizes `package.json` files. there is some noise in this commit because of the changes but I thought it was warranted.